File: fix.py
import os

file_path = r"c:\Users\jhxox\Desktop\sales_progerm\inphoto\gudok_PG\index.html"

with open(file_path, "r", encoding="utf-8") as f:
    lines = f.readlines()

new_lines = []
for i, line in enumerate(lines):
    line_num = i + 1
    if 1 <= line_num <= 720:
        continue
    if 988 <= line_num <= 1709:
        continue
    if 2323 <= line_num <= 2325:
        continue
    new_lines.append(line)

with open(file_path, "w", encoding="utf-8") as f:
    f.writelines(new_lines)

print("Done")

File: fix.js
const fs = require('fs');
const lines = fs.readFileSync('index.html', 'utf-8').split('\n');
const out = [];
for (let i = 0; i < lines.length; i++) {
    const num = i + 1;
    if (num >= 1 && num <= 720) continue;
    if (num >= 988 && num <= 1709) continue;
    if (num >= 2323 && num <= 2325) continue;
    out.push(lines[i]);
}
fs.writeFileSync('index.html', out.join('\n'));
console.log('Done!');
